src/main/java/org/folio/rest/impl/AutomatedPatronBlocksAPI.java

```diff
@@ -35,4 +45,63 @@
         GetAutomatedPatronBlocksByUserIdResponse.respond500WithTextPlain(
           failure.getLocalizedMessage()))));
   }
+
+  @Override
+  public void postAutomatedPatronBlocksSynchronizationRequest(SynchronizationRequest request,
+    Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    new SynchronizationRequestService(okapiHeaders, vertxContext.owner())
+      .createSyncRequest(request)
+      .onSuccess(response -> asyncResultHandler.handle(succeededFuture(
+        PostAutomatedPatronBlocksSynchronizationRequestResponse
+          .respond201WithApplicationJson(response))))
+      .onFailure(failure -> asyncResultHandler.handle(succeededFuture(
+        PostAutomatedPatronBlocksSynchronizationRequestResponse
+          .respond500WithTextPlain(failure.getLocalizedMessage()))));
```

Comment:
422 code - for example, when scope=user and userId is missing

Reply:
the validation has been added